main.rb

```diff
@@ -0,0 +1,87 @@
+class Main
+  library = Rental.new
+
+  puts 'What will you like to do? Choose an option.'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person ID'
+  puts '0. Quit'
+
+  choice = gets.chomp.to_i
+
+  case choice
+  when 0
+    break
+  when 1
+    library.list_books
+  when 2
+    library.list_people
+  when 3
+    puts 'What type of person do you want to create? Enter 1 for teacher or 2 student'
+    person_type = gets.chomp
+    case person_type
+    when 1
+      puts 'Enter Teachers Name'
+      name = gets.chomp
+      puts "Enter teacher's age"
+      age = gets.chomp.to_i
+      puts "Waht's teacher's specialization?"
+      specialization = gets.chomp
```

Comment:
- [ ] Good job so far 👏 When I try to run your program, ruby is flagging the syntax errors in the `main.rb` file. I tried to debug it, but every time it is showing another error. Please verify that it is working well and if you found out that something is wrong like syntax errors, please correct it. It is better if you could add procedures to run your program if they are specific in the README file. Here is the screenshot I took 👇 
![image](https://user-images.githubusercontent.com/90222110/219789197-ee6ee395-540b-401b-b44b-8481270f2358.png)